apps/server/src/modules/common-cartridge/controller/common-cartridge.controller.ts

```diff
@@ -13,4 +14,14 @@ export class CommonCartridgeController {
 	public async exportCourse(@Param() exportCourseParams: ExportCourseParams): Promise<CourseExportBodyResponse> {
 		return this.commonCartridgeUC.exportCourse(exportCourseParams.parentId);
 	}
+
+	@Get('export/lesson/:parentId')
+	public async exportLesson(@Param() exportCourseParams: ExportCourseParams): Promise<LessonDto> {
+		return this.commonCartridgeUC.exportLesson(exportCourseParams.parentId);
+	}
+
+	@Get('export/lessontasks/:parentId')

```

Comment:
Route should be `export/lesson/:id/tasks`.

Reply:
this is only for test that the Client works and should be deleted after review

Reply:
Ok, I will resolve this conservation when changes are reverted.